src/dashboard/Data/Playground/Playground.react.js

```
@@ -1,194 +1,1188 @@
+    updateSavedTabs(updatedTabs);
+    
+    // Save to localStorage
+    if (window.localStorage) {
+      try {
+        window.localStorage.setItem(tabsKey, JSON.stringify(updatedTabs));
+        window.localStorage.setItem(activeTabKey, nextTabId.toString());
+      } catch (e) {
+        console.warn('Failed to save tabs:', e);
+      }
+    }
+  }, [tabs, nextTabId, tabsKey, activeTabKey, updateSavedTabs]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)